src/main/java/duke/exceptions/UserInputException.java

```diff
@@ -0,0 +1,38 @@
+package duke.exceptions;
+
+import duke.Duke;
+public class UserInputException extends DukeException{
+    public enum inputExceptionType {
+        INVALID_TASK_TYPE, EMPTY_TASK_DESCRIPTION, EMPTY_DEADLINE_TIME,EMPTY_EVENT_START_TIME,EMPTY_EVENT_END_TIME
+    }
+    inputExceptionType exceptionType;
+    String exceptionMessage;
+    public UserInputException(inputExceptionType exceptionType) {
+        super("Exception of type: "+exceptionType+" occurred.");
```

Comment:
Missing spaces before and after the `+` sign here. Some similar issues can also be found in other lines. Do check them out!